### PR TITLE
FRW-8703 Fixed sitemap.

### PR DESCRIPTION
**Pull request by kraal-spryker**
## PR Description

Ticket: https://spryker.atlassian.net/browse/FRW-8703

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: sitemap.xml [hunk 1]
```diff
@@ -21,7 +19,7 @@ search: exclude
   {% unless page.search == "exclude" %}
   {% if site.url and page.url %}
```
Comment (dimitriyTsemma) [inline sitemap.xml:20]: Is this check still needed?

**Review by dimitriyTsemma — DISMISSED** [review]

File: sitemap.xml [hunk 2]
```diff
@@ -7,23 +7,19 @@ search: exclude
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
   {% for post in site.posts %}
   {% unless post.search == "exclude" %}
-  {% if site.url and post.url %}
   <url>
-    <loc>{{site.url}}{{post.url}}</loc>
```
Comment (asmarovydlo) [inline sitemap.xml:12]: Why those checks were removed?

Comment (kraal-spryker) [reply sitemap.xml:12]: they are not needed, they were introduced in my previous PR

Comment (asmarovydlo) [reply sitemap.xml:12]: Which one?

Comment (kraal-spryker) [reply sitemap.xml:12]: both

Comment (asmarovydlo) [reply sitemap.xml:12]: https://github.com/spryker/spryker-docs/pull/2809

**Review by asmarovydlo — APPROVED** [review]

**Pull request merged**